ibm_catalog.json

```diff
@@ -0,0 +1,146 @@
+{
+    "products": [
+        {
+            "features": [
+                {
+                    "description": "Supports multifactor authentication (MFA) for all users or federated users. Includes email-based, TOTP, and U2F MFA.",
+                    "title": "Multifactor authentication"
+                },
+                {
+                    "description": "Restrict API key creation (on-off)\nRestrict service ID creation (on-off)",
```

Comment:
We should get rid of the shorthand for the settings values. I don't think you need `on-off` at all in the description. 

And we can to keep the descriptions parallel with each other (Supports... Restricts..., Supports activity timeouts..., Sets the expiration of tokens..., Restricts access to IP addresses by..., 
`Restricts API key creation...)